### DISCUSSION TITLE
Please help create a collection

### DISCUSSION THREAD

**Discussion by abuturabofficial** [cotes2020/jekyll-theme-chirpy#1404] [Q&A]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

I want to create a `notes` collection which contains all my notes of different courses I have taken, with preferably having a nesting sub-collection of `notes/<course name>`. How to achieve it? Any help would be greatly appreciated.

### Operations you have already tried

I was able to make the collection and the layout of the individual note was of a post, but when I go to `mysite/notes` it lists all the notes files in the plain HTML.

How achieve this? I'm a total newbie and tried to follow the official docs, as well as copying some layouts of the `chirpy` theme and putting them in the `_layouts` dir, but to no avail. Can someone guide me or point me to the easy-to-understand guide?

### Anything else?

Currently, I don't have any screenshots as I tried to create collections almost a week ago, and got nowhere, then just gave up.

Today I again tried, by putting

```yaml
collections:
  notes:
    output: true
```
and created `_notes` dir, and also copied the `home.html` from chirpy layouts to the `_layouts/notes.html` with the front matter:
```yaml
---
layout: home
title: Notes
permalink: /notes/
---
```
I also deleted the original front matter of the `home.html`.

But today I wasn't even getting anything, but `404 error` when I went to `localhost:4000/notes`

![image](https://github.com/cotes2020/jekyll-theme-chirpy/assets/68259771/f759ab6d-137a-4dd6-bbbe-3b333980b3b0)

**Comment by jjnevis** [comment]
It's a shame you didn't get an answer to this, I'm in the same boat. It appears to be a very simple thing to do but difficult to achieve. I must be doing something basic wrong. I'll keep looking.

**Reply by SoumyaK4** [reply]
@jjnevis check my repo named SoumyaK4

**Reply by jjnevis** [reply]
I'm actually looking at your site (https://soumyak4.in/) just now, very nice! I'll have a look at your repo now, thanks!
